test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,37 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() {
+        // empty argument
```

Comment:
Minor nit. If you're writing comments starting with upper case, it's best to follow it throughout :)